flaskServer/restSample.py

```diff
@@ -12,18 +11,16 @@
 class uploadPhoto(Resource):
 	
 	def get(self):
-		return {'status': 'success'}
+		return {'server status': 'server is up and running'}
 		
 
 	def post(self):
 		self.reqparse = reqparse.RequestParser()
-		#self.reqparse.add_argument('picture', type=werkzeug.datastructures.FileStorage, location='files')
 		args = self.reqparse.parse_args()
 		f = request.files['photo']
```

Comment:
Maybe a more descriptive variable name? f was a bit ambiguous to me on first read.